app/src/organisms/ProtocolSetupModules/FixtureTable.tsx

```diff
@@ -55,6 +63,8 @@ export function FixtureTable({
         ]
       : []
 
+  const configuration = useLoadedFixturesConfigStatus(requiredFixtureDetails)
```

Comment:
The return type is `LoadedFixtureConfiguration[]` so probably configurations would be better.

```suggestion
  const configurations = useLoadedFixturesConfigStatus(requiredFixtureDetails)
```